### PR TITLE
KOGITO-2835: Online - Update 'edit existing file' to use the FileUpdate component

### PR DESCRIPTION
**Pull request by ljmotta**
## Task
https://issues.redhat.com/browse/KOGITO-2835
https://issues.redhat.com/browse/KOGITO-1517

### CODE REVIEW

File: packages/online-editor/src/home/HomePage.tsx [hunk 1]
```diff
@@ -578,38 +478,31 @@ export function HomePage(props: Props) {
...
+                  helperTextInvalid={i18n.homePage.uploadFile.helperInvalidText}
+                  validated={isUploadRejected ? "error" : "default"}
+                >
+                  <FileUpload
+                    id={"file-upload-field"}
+                    filenamePlaceholder={i18n.homePage.uploadFile.placeholder}
+                    filename={uploadedFileName}
+                    onChange={onFileUpload}
+                    dropzoneProps={{
+                      accept: ".dmn, .bpmn, .bpmn2",
```
Comment (tiagobento) [inline packages/online-editor/src/home/HomePage.tsx]: You could use the keys on the `editorEnvelopeLocator.mapping` object to generate this list and avoid duplication.

**Review by tiagobento — CHANGES_REQUESTED** [review]

File: packages/online-editor/src/home/HomePage.tsx (hunk 1 above)
Comment (ljmotta) [reply packages/online-editor/src/home/HomePage.tsx]: Done

**Review by tiagobento — APPROVED** [review]

File: packages/online-editor/src/home/HomePage.tsx [hunk 2]
```diff
@@ -26,6 +26,8 @@ import {
   Dropdown,
   DropdownItem,
   DropdownToggle,
+  FileUpload,
+  FileUploadField,
```
Comment (jomarko) [inline packages/online-editor/src/home/HomePage.tsx]: `FileUploadField` is not used

**Review by jomarko — COMMENTED** [review]
Question about different behavior. Once user click browse a upload file dialog is opened. This dialog has small selectbox in bottom right corner to filter files according an extension. 

In firefox selectbox offer:
- all files
- all supported files
- bpmn
- bpmn2
- dmn

in chrome selectbox offer:
- all files
- custom files

is this expected?

File: packages/online-editor/src/home/HomePage.tsx (hunk 2 above)
Comment (ljmotta) [reply packages/online-editor/src/home/HomePage.tsx]: Done

**Comment by ljmotta** [comment]
@jomarko Thanks for your review! Yes, it is expected. It depends of the browser implementation. ( It looks like the Firefox team has done a better job :grin: )

**Review by jomarko — APPROVED** [review]

**Review by ederign — APPROVED** [review]

**Pull request merged**